libs/utils/test.py

```diff
@@ -54,7 +54,7 @@ class LisaTest(unittest.TestCase):
     """Override this with a dictionary to configure the Executor"""
 
     @classmethod
-    def _init(cls, *args, **kwargs):
+    def init(cls, *args, **kwargs):
```

Comment:
Here perhaps I see a better usage for the currently empty runTest() method you added in the first patch.

Could it make sense to avoid this patch and call this _init() from the runTest()?
It would make also a more consistent interface from the Notebooks side... cause we will have:

```
from tests.eas.load_tracking import FreqInvarianceTest
t = FreqInvarianceTest()
r.runTests()
```

which makes more sense instead of calling a `t.init()` which is actually running tests.